### PR TITLE
M11 241 optimierung dropdowns

### PR DESCRIPTION
**Pull request by YoshiKanai**
Three Dropdowns in the menubar (site, language, currency) align to logged-in account menu on desktop. (mobile remain as default select-option from device.

### CODE REVIEW

File: src/components/Header/NavDropdown.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  list = [],
+  currentValue,
+  onChangeHandler = () => {},
+  children,
+}) {
+  const [isOpen, setIsOpen] = useState(false)
+  const selectRef = useRef()
+
+  const handleClick = (event, selected) => {
+    console.log('click ', selected)
```
Comment (SimonLudwigNeteleven) [inline src/components/Header/NavDropdown.js:15]: log statement

File: src/components/Header/NavDropdown.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          <HiChevronDown
+            size={20}
+            className="-mr-1 ml-1 h-5 w-5"
+            aria-hidden="true"
+          />
+        </span>
+      </button>
+      <div
+        className={`${
+          isOpen ? 'opacity-100' : 'invisible opacity-0'
```
Comment (SimonLudwigNeteleven) [inline src/components/Header/NavDropdown.js:50]: element will be rendered still but hidden visually. is that desired behaviour?

File: src/components/Header/NavDropdown.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        } absolute mt-1 min-w-full origin-top-right right-0 min-w-24 rounded-md shadow-lg bg-white ring-black ring-opacity-5 focus:outline-none transition-opacity duration-600`}
+      >
+        <div className="py-1">
+          {list &&
```
Comment (SimonLudwigNeteleven) [inline src/components/Header/NavDropdown.js:54]: would it not be better to put the condition for the list around the whole element like in line 48 for example?

Comment (SimonLudwigNeteleven) [reply src/components/Header/NavDropdown.js:54]: an maybe check also list length in addition to list

File: src/pages/account/addresses/AddressEdit.js [hunk 4]
```diff
@@ -189,7 +189,9 @@ const AddressesEdit = () => {
               />
             </div>
             <div className="col-span-1 flex flex-col">
-              <label className="mb-1">Country:</label>
+              <label htmlFor="" className="mb-1">
```
Comment (SimonLudwigNeteleven) [inline src/pages/account/addresses/AddressEdit.js:192]: what is htmlFor for?

File: src/components/Header/navigationbar.js [hunk 5]
```diff
@@ -1,319 +1,185 @@
...
+                    <option key={site.code} value={site.code}>
+                      {site.name}
+                    </option>
+                  )
+                })}
+            </select>
+          </div>
+        )}
+        <div
+          // className="w-full h-[59px] border-y flex justify-between items-center mt-6 font-inter text-base">
```
Comment (SimonLudwigNeteleven) [inline src/components/Header/navigationbar.js:162]: remove comment

File: src/components/Header/navigationbar.js [hunk 6]
```diff
@@ -1,319 +1,185 @@
...
+          <select className="text-tinBlue appearance-none">
+            <option value="Engish">English</option>
+            <option value="Italian">Italian</option>
+            <option value="French">French</option>
           </select>
         </div>
-        <div className="ml-[22px]">
-              <span className='hidden lg:inline'>{fields.currencyLabel}:</span>
+        <div
+          // className="w-full h-[59px] border-y flex justify-between items-center">
```
Comment (SimonLudwigNeteleven) [inline src/components/Header/navigationbar.js:173]: remove comment

File: src/components/Header/NavDropdown.js (hunk 2 above)
Comment (YoshiKanai) [reply src/components/Header/NavDropdown.js:50]: the logic is tablet without label, desktop with label, in order to win the space in menu bar. (mobile is rendered separate DOM)

File: src/components/Header/NavDropdown.js (hunk 3 above)
Comment (YoshiKanai) [reply src/components/Header/NavDropdown.js:54]: yes true, done

File: src/pages/account/addresses/AddressEdit.js (hunk 4 above)
Comment (YoshiKanai) [reply src/pages/account/addresses/AddressEdit.js:192]: removed not required this case.

**Pull request merged**